randovania/data/json_data/prime1.txt

```diff
@@ -196,207 +199,421 @@ Transport to Magmoor Caverns West
+          Plasma Beam and Space Jump Boots
+          Missile or Super Missile
 
 > Dock to Courtyard Entryway; Heals? False
-  * Normal Door to Courtyard Entryway/Dock to Ice Ruins West
+  * Wave Door to Courtyard Entryway/Dock to Ice Ruins West
+  > Dock to Ruins Entryway
+      Trivial
 
 > Dock to Canyon Entryway; Heals? False
-  * Normal Door to Canyon Entryway/Dock to Ice Ruins West
+  * Missile Blast Shield to Canyon Entryway/Dock to Ice Ruins West
+  > Dock to Ruins Entryway
+      Trivial
+
+> Pickup (Power Bomb); Heals? False
+  * Pickup 38; Major Location? False
+  > Dock to Ruins Entryway
+      Trivial
 
 ----------------
 Ice Ruins East
 Asset id: 3673997935
 > Dock to Ice Ruins Access; Heals? False
   * Normal Door to Ice Ruins Access/Dock to Ice Ruins East
+  > Dock to Plaza Walkway
+      Space Jump Boots
+  > Pickup (Spider Track Missile Expansion)
+      Morph Ball and Spider Ball
+  > Pickup (Missile Expansion Behind Ice)
+      Plasma Beam
 
 > Dock to Plaza Walkway; Heals? False
   * Normal Door to Plaza Walkway/Dock to Ice Ruins East
+  > Dock to Ice Ruins Access
+      Trivial
+
+> Pickup (Spider Track Missile Expansion); Heals? False
+  * Pickup 40; Major Location? False
+  > Dock to Ice Ruins Access
+      Trivial
+
+> Pickup (Missile Expansion Behind Ice); Heals? False
+  * Pickup 39; Major Location? False
+  > Dock to Ice Ruins Access
+      Trivial
 
 ----------------
 Chapel Tunnel
 Asset id: 4016524108
 > Dock to Chapel of the Elders; Heals? False
   * Normal Door to Chapel of the Elders/Dock to Chapel Tunnel
+  > Dock to Chozo Ice Temple
+      Morph Ball Bomb and Morph Ball
 
 > Dock to Chozo Ice Temple; Heals? False
   * Normal Door to Chozo Ice Temple/Dock to Chapel Tunnel
+  > Dock to Chapel of the Elders
+      Morph Ball Bomb and Morph Ball
 
 ----------------
 Courtyard Entryway
 Asset id: 3484986321
 > Dock to Ruined Courtyard; Heals? False
   * Normal Door to Ruined Courtyard/Dock to Courtyard Entryway
+  > Dock to Ice Ruins West
+      Trivial
 
 > Dock to Ice Ruins West; Heals? False
   * Normal Door to Ice Ruins West/Dock to Courtyard Entryway
+  > Dock to Ruined Courtyard
+      Trivial
 
 ----------------
 Canyon Entryway
 Asset id: 55410999
 > Dock to Phendrana Canyon; Heals? False
   * Normal Door to Phendrana Canyon/Dock to Canyon Entryway
+  > Dock to Ice Ruins West
+      Trivial
 
 > Dock to Ice Ruins West; Heals? False
-  * Normal Door to Ice Ruins West/Dock to Canyon Entryway
+  * Missile Blast Shield to Ice Ruins West/Dock to Canyon Entryway
+  > Dock to Phendrana Canyon
+      Trivial
 
 ----------------
 Chapel of the Elders
 Asset id: 1086671081
 > Dock to Chapel Tunnel; Heals? False
-  * Normal Door to Chapel Tunnel/Dock to Chapel of the Elders
+  * Wave Door to Chapel Tunnel/Dock to Chapel of the Elders
+  > Pickup (Wave Beam)
+      Any of the following:
+          Missile
+          All of the following:
+              Morph Ball
+              Morph Ball Bomb or Power Bomb
```

Comment:
I'm not sure how many Missiles would be safe here. Probably better to just require Bombs. PBs would be a beginner trick.